com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalTarget.cs

```diff
@@ -523,6 +643,17 @@ static class CoreRenderStates
             { RenderState.Blend(Blend.One, Blend.One, Blend.One, Blend.One), new FieldCondition(UniversalFields.BlendAdd, true) },
             { RenderState.Blend(Blend.DstColor, Blend.Zero), new FieldCondition(UniversalFields.BlendMultiply, true) },
         };
+
+        public static readonly RenderStateCollection SceneSelection = new RenderStateCollection
+        {
+            { RenderState.Cull(Cull.Off) },
+        };
+
+        public static readonly RenderStateCollection ScenePicking = new RenderStateCollection
+        {
+            { RenderState.Cull(Cull.Back), new FieldCondition(Fields.DoubleSided, false) },
```

Comment:
This should likely be changed once @cdxntchou lands his Uber shader change here: https://github.com/Unity-Technologies/Graphics/pull/3705.
Mostly a note for myself depending on which PR wins.
This just needs to be updated to use the property the same way that HDRP does.